wavefront_lambda/wavefront_lambda/__init__.py

```diff
@@ -0,0 +1,112 @@
+from wavefront_pyformance.wavefront_reporter import WavefrontDirectReporter
+from pyformance import MetricsRegistry
+import os
+from datetime import datetime
+from wavefront_pyformance import delta
+
+is_cold_start = True
+reg = None
+
+
+def wrapper(func):
+    """
+    Returns the Wavefront Lambda wrapper. The wrapper collects aws lambda's
+    standard metrics and reports it directly to the set wavefront url. It
+    requires the following Environmental variables to be set:
+    1.WAVEFRONT_URL : your corp wavefront url
+    2.WAVEFRONT_API_TOKEN : The users API authentication token
```

Comment:
Same as above, mention Direct Data Ingestion permission is required for the associated token.